subprojects/core/src/main/java/org/gradle/invocation/DefaultGradle.java

```diff
@@ -114,7 +114,7 @@ public void projectsLoaded(Gradle gradle) {
                 if (!rootProjectActions.isEmpty()) {
```

Comment:
❓  `projectsLoaded` is guaranteed to be called only once for this `Gradle` instance during the entire build, right?

(wondering whether `IPELP.isolateFor(...)` needs to be concurrency-safe)

Reply:
Yes.